Resources/doc/index.rst

```diff
@@ -41,13 +41,36 @@ application:
 
     # config/packages/doctrine_migrations.yaml
     doctrine_migrations:
+        name: 'Application Migrations'
+
+        # Namespace/path to search for migrations
+        migrations_paths:
+            'DoctrineMigrations': '%kernel.project_dir%/src/Migrations'
+
+        # Deprecated since v2.2, use "migrations_paths" instead (array value)
         dir_name: '%kernel.project_dir%/src/Migrations'
+
+        # Deprecated since v2.2, use "migrations_paths" instead (array key)
         namespace: DoctrineMigrations
+
+        storage:
+            # Default (SQL table) metadata storage configuration
+            table_storage:
+                table_name: 'migration_versions'
+                version_column_name: 'version'
+                version_column_length: 1024
+                executed_at_column_name: 'executed_at'
+                execution_time_column_name: 'execution_time'
```

Comment:
Using these as default values would spare you from all the `isset` checks, and be maybe more friendly to someone using `debug:config` or `config:dump-reference`